lib/widget/input/form_textfield.dart

```diff
@@ -199,9 +201,23 @@ class TextInputController extends FormFieldController {
   String? mask;
   int maxLines = 1;
   TextStyle? textStyle;
+  LabelStyle? labelStyle;
   TextStyle? hintStyle;
 }
 
+enum LabelPosition { outline, inline }
+
+class LabelStyle extends TextStyle {
+  TextStyle? textStyle;
+  LabelPosition position = LabelPosition.outline;
```

Comment:
don't overload TextStyle. From the YAML perspective, the user only understand TextStyle. It will be super confusing If we add a position attribute ONLY for form label and nowhere else.
I think you can use boolean `floatLabel` at the style root.

Reply:
Okay vu, I'll change then and make the `floatLabel` defaults to false

```yaml
styles:
  {
    label: '${emailId.value.length == 0 ? "" : "Name"}',
    floatLabel: true, # Defaults to false
    labelStyle:
      {
        fontWeight: semibold,
        fontSize: 34,
        color: green,
      },
  }
```

Reply:
@vusters Can you review it again? I updated the PR